packages/postcss-zindex/src/lib/layerCache.js

```diff
@@ -1,33 +1,16 @@
-import has from 'has';
-import uniq from 'uniqs';
-
-function LayerCache(opts) {
-  this._values = [];
-  this._startIndex = opts.startIndex || 1;
+function LayerCache() {
+  this._values = new Map();
 }
 
 function ascending(a, b) {
   return a - b;
 }
 
-function reduceValues(list, value, index) {
-  list[value] = index + this._startIndex;
-
-  return list;
-}
-
-LayerCache.prototype._findValue = function (value) {
```

Comment:
This whole method is not needed anymore, since we can access the map by key directly.